charts/illumidesk/templates/grader_setup_service.yaml

```diff
@@ -107,14 +107,23 @@ spec:
           env:
             - name: GRADER_IMAGE_NAME
               value: {{ .Values.graderSetupService.graderImage }}
-            - name: POSTGRES_NBGRADER_HOST
+            - name: POSTGRES_GRADER_SETUP_HOST
               value: {{ .Values.graderSetupService.postgresNBGraderHost }}
-            - name: POSTGRES_NBGRADER_USER
+            - name: POSTGRES_GRADER_SETUP_USER
               value: {{ .Values.graderSetupService.postgresNBGraderUser | default "postgres" }}
-            - name: POSTGRES_NBGRADER_PASSWORD
+            - name: POSTGRES_GRADER_SETUP_PASSWORD
```

Comment:
@Abhi94N I'm not sure this change is necessary. The reason they were initially named with the `POSTGRES_NBGRADER_*` prefix is to explicitly identify that these environment variables correspond to the shared grader notebook settings that the `grader-setup-service` uses when creating a new deployment. These names may conflict with the environment variables that we could add in the future if we add Postgres support for the `grader-setup-service`.

Reply:
would @ashwinm-tt need to update the envars for the his docker file and grader setup service code base repo?

Reply:
updated back to NBGRADER